FlashCap.Core/Internal/NativeMethods.cs

```diff
@@ -198,7 +198,8 @@ public enum Compression
         YUY2 = 0x32595559,   // FOURCC
         YUYV = 0x56595559,   // FOURCC
         UYVY = 0x59565955,   // FOURCC
-        MJPG = 0x47504A4D,   // FOURCC
+        HDYC = 0x43594448,   // FOURCC
```

Comment:
The comment:

```csharp
// FOURCC (BlackMagic SDI input (UYVY))
```

so that it can be identified as a special FOURCC.
I am not familiar with BlackMagic's products, but if you know it works for non-SDI products, you can leave out the `SDI input`.